src/lib/components/scheduling/ScheduleTimeline.svelte

```diff
@@ -82,12 +80,6 @@
 </Row>
 
 <style>
-	.vertical-text {
-		writing-mode: vertical-rl;
-		text-orientation: mixed;
-		margin-left: -10px;
-	}
-
```

Comment:
why no vertical?

Reply:
Takes up more space and imo is less readable